### PR TITLE
[evas] Change EAPI symbols for EVAS_API

### PR DESCRIPTION
**Pull request by Coquinho**

### CODE REVIEW

File: src/lib/evas/include/evas_common_private.h [hunk 1]
```diff
@@ -1151,7 +1151,7 @@ EVAS_API void generic_cache_data_drop(Generic_Cache *cache, void *key);
 }
 #endif
 
-#undef EAPI
-#define EAPI
+#undef EVAS_API
+#define EVAS_API
```
Comment (joaoantoniocardoso) [inline src/lib/evas/include/evas_common_private.h:1155]: Is it really necessary?

File: src/lib/evas/include/evas_private.h [hunk 2]
```diff
@@ -1577,8 +1552,8 @@ _gfx_to_evas_render_op(Efl_Gfx_Render_Op rop)
 }
 #endif
 
-#undef EAPI
-#define EAPI
+#undef EVAS_API
```
Comment (joaoantoniocardoso) [inline src/lib/evas/include/evas_private.h:1555]: I think this _undef define_ should be removed to the ensure the same definition for `EVAS_API`  to be used in the files that is using this one as headers.

File: src/lib/evas/include/evas_common_private.h (hunk 1 above)
Comment (Coquinho) [reply src/lib/evas/include/evas_common_private.h:1155]: O.o

Comment (Coquinho) [reply src/lib/evas/include/evas_common_private.h:1155]: working on it

**Comment by Coquinho** [comment]
There were changes, so please look again.

**Pull request merged**